src/main/java/com/cloudbees/jenkins/GitHubWebHook.java

```diff
@@ -153,7 +157,10 @@ public static Jenkins getJenkinsInstance() throws IllegalStateException {
      * Other plugins may be interested in listening for these updates.
      *
      * @since 1.8
+     * @deprecated we do not think this API is required any more, if we are wrong, please raise a JIRA
      */
+    @Deprecated
+    @Restricted(NoExternalUse.class)
```

Comment:
[the list](https://wiki.jenkins-ci.org/display/JENKINS/Extension+points#Extensionpoints-com.cloudbees.jenkins.GitHubWebHook.Listener)